README.md

```diff
@@ -18,6 +18,8 @@ Installers and binaries for both Windows and Linux are available from [our websi
 
 Development builds are also available from [our website](https://pcsx2.net/download/development.html).
 
+For compilation instructions please follow our [compilation guide](COMPILING.md).
```

Comment:
Might be nice to mention that it's for Linux distros in general. What if a windows user sees this line?

Reply:
this is why i said we will probably add windows too in this PR, so both guides are available at the same place.